test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -136,15 +136,15 @@ void test_ucp_tag::wait_and_validate(request *req)
     request_release(req);
 }
 
-void test_ucp_tag::wait_for_unexpected_msg(ucs_queue_head_t unexpected_q,
+void test_ucp_tag::wait_for_unexpected_msg(ucs_queue_head_t *unexpected_q,
```

Comment:
it was fixed once, but then the fault was introduced again with new wait_for_unexpected_msg method